sqlitecluster/SQLiteNode.cpp

```diff
@@ -862,6 +864,16 @@ bool SQLiteNode::update() {
             }
         }
 
+        // If the majority of full peers responds with abstain, then re-search.
+        const bool majorityAbstained = abstainCount * 2 >= numFullPeers;
```

Comment:
If we have 4 full peers (our typical case), and two abstain, then this will say that the majority has abstained, when we still have ourself, and 2 other peers that have not abstained, which is enough for a full cluster.

I think this needs to be `>` not `>=`.

Reply:
Right, that makes sense, updated